dir.traversal.targets

```diff
@@ -2,7 +2,8 @@
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Target Name="FilterProjects">
     <PropertyGroup>
-      <FilterToOSGroup Condition="'$(FilterToOSGroup)'=='' and '$(OSGroup)'!='' and '$(OSGroup)'!='AnyOS'">$(OSGroup)</FilterToOSGroup>
```

Comment:
Perhaps condition this on BuildAllOSGroups as well instead of commenting it out.